integrations/bytewax/README.md

```diff
@@ -0,0 +1,86 @@
+# Real time feature computation using Bytewax.
+
+## Introduction
+In this guide you will learn how to create Real-Time Feature Engineering pipeline and write real time features in to
+the Hopsworks features store. This guide covers creating a
+
+- computing real time features with Bytewax.
+- writing real time features to Hopsworks's online feature.
+
+You will also
+- create feature group using the HSFS APIs.
+- Backfill feature data to offline feature group.
+
+## Before you begin
+For the tutorials to work, you need:
+- Hopsworks account
+[managed.hopsworks.ai](https://managed.hopsworks.ai) account or on premise Hopsworks deployment. Note that this tutorial 
+will not work for [app.hopsworks.ai](https://app.hopsworks.ai) account as submitting custom jobs to 
+[app.hopsworks.ai](https://app.hopsworks.ai) are not supported.
+
+You can find documentation how to get started on [GCP](https://docs.hopsworks.ai/3.1/setup_installation/gcp/getting_started/),
+[AWS](https://docs.hopsworks.ai/3.1/setup_installation/aws/getting_started/) or on [Azure](https://docs.hopsworks.ai/3.1/setup_installation/azure/getting_started/).
+
+Install required python libraries 
+```bash
+pip install hopsworks
+pip install bytewax
+pip install websocket-client
+```
+
+you also need  to have Hopsworks cluster host address, hopsworks project name and 
+[api key](https://docs.hopsworks.ai/3.1/user_guides/projects/api_key/create_api_key/)
+
+Once you have the above define environment variables:
+
+```bash
+export HOPSWORKS_HOST=REPLACE_WITH_YOUR_HOPSWOKRKS_CLUSTER_HOST
+export HOPSWORKS_API_KEY=REPLACE_WITH_YOUR_HOPSWORKS_API_KEY
+export HOPSWOERKS_PROJECT_NAME=REPLACE_WITH_YOUR_HOPSWOERKS_PROJECT_NAME
+```
+
+## Clone tutorials repository
+```bash
+git clone https://github.com/logicalclocks/hopsworks-tutorials
+cd ./hopsworks-tutorials/integrations/bytewax
+```
+
+## Create Feature Groups
+Currently, bytewax support for Hopsworks feature store is experimental and only write operation is supported. This means
+that Feature group metadata needs to be registered in Hopsworks Feature store before you can write real time features computed
+by Bytewax.
```

Comment:
I would be interested to know what is needed to make the metadata on the fly. Could this be added to a custom output connector that inherits from the Kafka  Output used and creates the metadata on `__init__`.

We are adding a custom connector repository (early stages) with partner connectors and this feels like a nice one to eventually include.